quipucords/api/scanjob/model.py

```diff
@@ -302,31 +299,34 @@ def validate_status_change(self, target_status, valid_current_status):
             return True
         return False
 
-    def create_product_status_dict(self):
+    def create_extra_vars(self):
         """Construct a dictionary based on the disabled products.
 
         :returns: a dictionary representing the updated collection
-        status of the optional products
+        status of the optional products to be assigned as the extra
+        vars for the ansibile task runner
         """
         # Grab the optional products status dict and create
         # a default dict (all products default to True)
         product_status = self.get_optional_products_args()
```

Comment:
maybe just rename to `self.get_optional_products_args()` --> `self.get_optional_products()`